pkg/apis/hive/v1alpha1/syncset_types.go

```diff
@@ -68,6 +68,12 @@ type SyncObjectPatch struct {
 	// +optional
 	Namespace string `json:"namespace,omitempty"`
 
+	// PatchApplyMode indicates if the patch apply mode is "AlwaysApply" (default) or "ApplyOnce".
+	// PatchApplyode "AlwaysApply" indicates that the patch should be applied every time reconcilation occurs.
+	// PatchApplyMode "ApplyOnce" indicates that the patch should only be applied once.
+	// +optional
+	PatchApplyMode SyncSetPatchApplyMode `json:"mode,omitempty"`
```

Comment:
The go variable name should match the json name. In this case, it should be ok to just name it Mode or ApplyMode since the context is a patch.